journal/journal/models.py

```diff
@@ -0,0 +1,33 @@
+import datetime 
+
+from sqlalchemy import (
+    Column,
+    Index,
+    Integer,
+    Text,
+    Unicode,
+    UnicodeText,
+    DateTime,
+    )
+
+from sqlalchemy.ext.declarative import declarative_base
+
+from sqlalchemy.orm import (
+    scoped_session,
+    sessionmaker,
+    )
+
+from zope.sqlalchemy import ZopeTransactionExtension
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+Base = declarative_base()
+
+
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = Column(Integer, primary_key=True)
+    title = Column(Unicode(length=128), unique=True, nullable=False)
+    text = Column(UnicodeText)
+    created = Column(DateTime(), default=datetime.datetime.utcnow())
```

Comment:
All your entries will be created with the exact same `created` date if you keep this setup.  Remember, you want to pass a callable function as the `default` argument to the column constructor, not the result of calling that callable.